.github/pull_request_template.md

```diff
@@ -1,18 +1,21 @@
 <!--
-Thank you for your pull request.
+Thank you for your pull request!
+
 Please see http://site.icu-project.org/processes/contribute for general
 information on contributing to ICU.
 
-You will be automatically asked to sign the contributors license before the PR is accepted.
+You will be automatically asked to sign the contributors license (CLA) before the PR is accepted.
 - sign: https://cla-assistant.io/unicode-org/icu
 - license: http://www.unicode.org/copyright.html
+
+Please note that all commit messages must be prefixed with a JIRA Issue number.
```

Comment:
Let's do this as a checklist item, see comment below. I copied/modified your addition.